src/main/java/org/spongepowered/api/entity/living/golem/IronGolem.java

```diff
@@ -36,4 +36,8 @@
         return getValue(Keys.PLAYER_CREATED).get();
     }
 
+    default Value<Boolean> isHoldingFlower() {
+        return getValue(Keys.IRON_GOLEM_HOLDING_FLOWER).get();
```

Comment:
This is a field in iron golem's vanilla code which stores this value, though it has no usage in the class itself, as it only has getters and setters.